test/functional/tools/system/pip_manager_test.py

```diff
@@ -0,0 +1,123 @@
+import textwrap
+from conan.test.utils.tools import TestClient
+from conan.internal.util.files import save_files
+from conan.test.utils.test_files import temp_folder
+
+
+def _create_py_hello_world(folder):
+    setup_py = textwrap.dedent("""
+        from setuptools import setup, find_packages
+
+        setup(
+            name='hello',
+            version='0.1.0',
+            packages=find_packages(include=['hello', 'hello.*']),
+            entry_points={'console_scripts': ['hello-world = hello:hello']}
+        )
+        """)
+    hello_py = textwrap.dedent("""
+        def hello():
+            print("Hello Test World!")
+        """)
+
+    save_files(folder, {"setup.py": setup_py, "hello/__init__.py": hello_py})
+
+
+def test_build_pip_manager():
+
+    pip_package_folder = temp_folder(path_with_spaces=False)
+    _create_py_hello_world(pip_package_folder)
+    pip_package_folder = pip_package_folder.replace('\\', '/')
+
+    conanfile_pip = textwrap.dedent(f"""
+        from conan import ConanFile
+        from conan.tools.system.pip_manager import Pip
```

Comment:
```suggestion
        from conan.tools.system import Pip
```
Imports from tools must be from this level, other internal level are private